misc/config_definitions.json

```diff
@@ -864,7 +864,16 @@
         },
         "default_port_group": {
             "default": 0,
-            "type": "integer"
+            "type": "select-dynamic",
+            "group": "discovery",
+            "section": "networks",
+            "order": 0,
+            "options": {
+                "target": "port-group"
+            },
+            "validate": {
+                "value": "integer|zero_or_exists:port_groups,id"
```

Comment:
You need to inject the definition for 0 on the PortGroupController.

Reply:
how can i do it in an elegant way?

Reply:
injected